src/app/core/components/add-question-form/add-question-form.component.html

```diff
@@ -0,0 +1,72 @@
+<div>
+  <h2 style="text-align: center; margin-top: 20px">Додайте, будь ласка, своє питання</h2>
+  <form [formGroup]="questionForm" (ngSubmit)="addQuestion()">
+
+    <div class="form">
+      <mat-form-field>
+        <mat-label>Виберіть предмет</mat-label>
+        <mat-select formControlName="subject" [errorStateMatcher]="matcher">
+          <mat-option *ngFor="let sub of subjects" [value]="sub.value">
+            {{sub.viewValue}}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>Виберіть тему</mat-label>
+        <mat-select formControlName="theme" [errorStateMatcher]="matcher">
+          <mat-option *ngFor="let theme of themes" [value]="theme.value">
+            {{theme.viewValue}}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>Виберіть складність</mat-label>
+        <mat-select formControlName="level" [errorStateMatcher]="matcher">
+          <mat-option *ngFor="let level of levels" [value]="level.value">
+            {{level.viewValue}}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field class="example-full-width">
+        <input matInput placeholder="Вкажіть теги для питання" formGroupName="tags"
+               [errorStateMatcher]="matcher" (input)="newTag($event)">
+      </mat-form-field>
+
+      <mat-form-field class="example-full-width">
+        <input matInput placeholder="Сформулюйте питтаня" formControlName="question"
+               [errorStateMatcher]="matcher">
+      </mat-form-field>
+    </div>
+
+    <button mat-raised-button
+            color="primary"
+            type="button"
+            (click)="addAnswer()"
+            [disabled]="questionForm.get('answersForm').controls.length > 4">
```

Comment:
Maybe it will be better to replace 4 to constants?

Reply:
fixed